pkg/testing/matrix/matrix_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2016-2021, Pulumi Corporation.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package matrix
+
+import (
+	"os/exec"
+	"testing"
+
+	"github.com/blang/semver"
+	i "github.com/pulumi/pulumi/pkg/v3/testing/integration"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/workspace"
+)
+
+func TestAll(t *testing.T) {
+	t.Parallel()
+	opts := []TestOptions{
+		/*{
```

Comment:
Can these all be enabled now?

Reply:
Stack references still cause the whole thing to panic, otherwise yes